### DISCUSSION TITLE
I haven't yet received the benefits of the GitHub student Developer Pack over last 5 months.

### DISCUSSION THREAD

**Discussion by Arlene-1998** [community/community#131812] [GitHub Education]
### Hi everyone,

I haven't yet received the benefits of the GitHub student Developer Pack since Jan 20.
<img width="322" alt="image" src="https://github.com/community/community/assets/41280952/671076b7-92be-4430-a707-2129cf4c5e94">

**Comment by paiineli** [comment]
Hi,

I understand your frustration regarding the delay in receiving the benefits of the GitHub Student Developer Pack since January 20. Here are a few steps you can take:

Check Your Email: Ensure you have received any confirmation emails related to your application. Sometimes additional steps are needed to activate your benefits.

Log Out and Back In: Try logging out of your GitHub account and logging back in to see if the benefits appear.

Verify Your Educational Email: Make sure your educational email is verified and linked to your GitHub account.

Contact GitHub Support: If the issue persists, I recommend reaching out to GitHub Support directly. Provide them with the details of your application for quicker assistance.

I hope this gets resolved soon!

Best,

**Comment by samus-aran** [comment]
Thanks @Arlene-1998 👋 

We've passed this along to our internal team. We'll let you know if there are any updates here. 👍
